lib/Mojolicious/Plugin/DefaultHelpers.pm

```diff
@@ -90,6 +90,11 @@ sub _content {
   return Mojo::ByteStream->new($hash->{$name} // '');
 }
 
+sub _convert_to_exception {
+  my $e = shift;
+  return _is_e($e) ? $e : Mojo::Exception->new($e);
```

Comment:
Not sure if this is very important, but could `_is_e()` be inlined in this function? Looks like it's only being used one place now.

Reply:
I'll make that change right away, thanks for the feedback!

Reply:
Did _is_e() come back by accident?

Reply:
Must have screwed up my commit whoops